nextjs-metrics-viewer/app/(protected)/upload/Components/Modal/DeleteData.tsx

```
@@ -0,0 +1,34 @@
+import { Modal, Button } from "flowbite-react";
+
+interface props {
+  id: number;
+  open: boolean;
+  onClose: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)